### DISCUSSION TITLE
Best approach to ingest all docker containers logs into openobserve

### DISCUSSION THREAD

**Discussion by accforgithubtest** [openobserve/openobserve#2746] [Q&A]
I have just discovered openobserve and would like to give this a go with my self-hosted set up.

I am seeking some help as to what the best approach to ingest logs into openobserve is ? 

The stack on which I will be attempting this is a simple docker compose based stack (not swarm) with multiple containers all on the same host.

TIA for your suggestions !

**Comment by prabhatsharma** [comment]
Depending on what platform you are using, your docker logs will be stored in different folders.

Mac - ~/Library/Containers/com.docker.docker/Data/log
Linux - /var/lib/docker/containers
windows - C:\ProgramData\Docker\log

You can use a log forwarder (vector, fluentbit, otel-collector) to watch these folders and send them to OpenObserve

**Comment by accforgithubtest** [comment]
Thanks for your quick reply @prabhatsharma !
Will try your suggestions above and come back if I need more help. Cheers !
